guides/release/testing/testing-tools.md

```diff
@@ -0,0 +1,185 @@
+Ember comes with great testing tools out of the box and popular addons which you can use as your testing game evolves.
+
+In this section, we will go through various tools, that you can use while building an Ember app.
+
+## What Tools Can Help Me?
+
+### QUnit, QUnit DOM
+
+Every Ember app comes with [QUnit](http://qunitjs.com/) and [QUnit DOM](https://github.com/simplabs/qunit-dom). QUnit is a testing framework, and QUnit DOM is a library that helps you **write tests that are concise and readable**.
+
+To see the power of QUnit DOM, consider this code snippet. It checks whether our button component shows the right label and the right attributes.
+
+```javascript {data-filename=tests/integration/components/simple-button-test.js}
+/*
+  For simplicity, the import, module, and setup statements
+  are omitted here. Our component accepts two arguments,
+  label (string) and isDisabled (boolean).
+*/
+test("should show label", async function (assert) {
+  await render(hbs`
```

Comment:
is this hbs' a random name if yes can you give some valid name ?

Reply:
hbs (Handlebars) is a file extension similar to `.jsx` and `.tsx`
https://guides.emberjs.com/v3.3.0/templates/handlebars-basics/